sdk/keyvault/azure-security-keyvault-certificates/src/private/certificate_serializers.hpp

```diff
@@ -109,4 +109,13 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
 
       static std::string Serialize(CertificateIssuer const& issuer);
     };
+
+    class CertificateContactsSerializer final {
+      CertificateContactsSerializer() = delete;
```

Comment:
Can you help explain why we are deleting this? Maybe with a source comment?

Reply:
Victor and Anton said so, i'm just a humble developer, they wanted to prevent creating an instance of the serializer objects for some reason.